README.md

```diff
@@ -106,6 +106,26 @@ You can customize this in your config by specifying `xmlOptions`:
 }
 ~~~
 
+## Connector Options
+
+- `connectionTimeout` - Service-level Timeout. Represents integer containing the number of milliseconds to wait for a server to send response headers (and start the response body) before aborting the request. Note that if the underlying TCP connection cannot be established, the OS-wide TCP connection timeout will overrule the timeout option
+
+~~~
+{connectionTimeout: 5000}
+~~~
+
+You can customize this in your config file:
```

Comment:
Can you rephrase this to "You can customize the connector by adding any of the following options to your configuration file" and move the whole block with the config file example to the top under the Customizing header? I think then it is more clear for the user where to put all this.